src/main/java/edu/escuelaing/ieti/matchmaking/persistence/EquipoRepository.java

```diff
@@ -2,17 +2,18 @@
 
 import java.util.List;
 
+import edu.escuelaing.ieti.matchmaking.exception.MatchmakingException;
 import edu.escuelaing.ieti.matchmaking.model.Equipo;
 
 public interface EquipoRepository {
 	
-	Equipo create(Equipo equipo);
+	Equipo create(Equipo equipo) throws MatchmakingException;
```

Comment:
what is the purpose of throwing this checked exception here? how are you planning to catch it an handle it?